src/extension.ts

```diff
@@ -1425,7 +1425,7 @@ export class ExtensionManager implements vscode.Disposable {
             return false;
         }
         for (const project of projects) {
-            if (await project.hasCMakeLists()) {
+            if (project.hasCMakeLists()) {
```

Comment:
If the `await` is gone, consider removing the `async` specifier from this function.

Reply:
You could also use `return projects.some(p => p.hasCMakeLists());` instead of a for-loop here too.